Cargo.toml

```diff
@@ -29,10 +29,10 @@ winapi = "0.1.17"
 kernel32-sys = "0.1.0"
 
 [dev-dependencies.glium]
-version = "0.2.1"
+version = "^0.3"
 default-features = false
 features = ["gl_read_buffer", "gl_depth_textures"]
 
 [dev-dependencies.cgmath]
+version = "^0.1"
 git = "https://github.com/bjz/cgmath-rs"
```

Comment:
I'm still learning the idiosyncrasies if the cargo semver numbers so I was guessing at the best way to set the correct updating while making minimal changes to your Cargo.toml. I used the ^0.x form because supposedly minor versions within 0.x are not API stable and so until 1.0 when new 0.x versions of the dependencies are released they *could* break the code again, but patches should be safe.